charts/aws-pca-issuer/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: aws-privateca-issuer
 description: An addon for cert-manager to sign certificates using AWS PCA
 type: application
-version: v1.2.5
```

Comment:
This version bump is for when we want to cut a new release. Let's keep this at 1.2.5 for now